### ISSUE TITLE
LDAP Users have the Standard User role assigned, which is not always the right choice

### ISSUE THREAD

**Issue by lonewulf** [rancher/rancher#13751]
**Rancher versions:**
rancher/server or rancher/rancher: 2.0.2
rancher/agent or rancher/rancher-agent: 2.0.2

**Steps to Reproduce:**
Login with an LDAP user, that user can create cluster (Though it can't create tokens to add nodes)

**Results:**
LDAP users are set as 'Standard User'.

**Expected Behavior:**
There should be a mean to specify which permission a new LDAP user should have upon first login.

**Comment by cjellick** [comment]
Agreed! This is something we are working on. I'm going to close this as a duplicate of https://github.com/rancher/rancher/issues/12737
Though I have one follow up question: what role would you make the new default? Or more specifically, what would you want the user to be able to do/not do?
Can you post your answer in the other issue?
